source/features/file-finder-buffer.tsx

```diff
@@ -0,0 +1,53 @@
+import './file-finder-buffer.css';
+import React from 'dom-chef';
+import features from '../libs/features';
+import select from 'select-dom';
+
+const FILE_FINDER_INPUT_SELECTOR = '.js-tree-finder > .breadcrumb > #tree-finder-field';
+
+const pjaxStartHandler = ((event: CustomEvent): void => {
+	const destinationURL: string = event.detail?.url || '';
+	if (destinationURL.includes('/find/')) {
+		const hiddenInput = select<HTMLInputElement>('#rgh-file-finder-buffer') ?? document.body.insertBefore(
```

Comment:
This was intentionally written so that instead of adding/removing it every time, we could continue using the input field once it is created. The same reason why I chose to empty it in pjax:complete instead of removing it.